optimum/habana/transformers/models/persimmon/modeling_persimmon.py

```diff
@@ -391,8 +391,7 @@ def forward(
         )
 
         hidden_states = outputs[0]
-        # No upscaling to float was ever done for Persimmon
-        logits = self.lm_head(hidden_states[:, -num_logits_to_keep:, :])
+        logits = self.lm_head(hidden_states)
 
```

Comment:
This model doesn't seem to have trim_logits, we can notify the author to add the trim logits support.